controllers/nodeagent_test.go

```diff
@@ -3182,3 +3183,119 @@ func TestDPAReconciler_updateFsRestoreHelperCM(t *testing.T) {
 		})
 	}
 }
+
+func TestDPAReconciler_getPlatformType(t *testing.T) {
```

Comment:
since there is only one test scenario, I would simplify and not do a range

Reply:
added another test case